src/js/modules/RoutingApp.js

```diff
@@ -17,51 +22,62 @@ const RoutingApp = ({ sections, roles }) => {
     return Object.keys(sections).map(sectionSlug => {

```

Comment:
No problem. The reason is the `<Swirdth>` breaking with the `<div>`'s inside thing.